.github/dependabot.yml

```diff
@@ -10,3 +10,20 @@ updates:
       github-actions:
         patterns:
           - "*"
+  - package-ecosystem: "gitsubmodule"
+    directory: "/"        
+    schedule:
+      interval: "daily"
+    target-branch: "main"
+    allow:
+      - dependency-name: "half"
+      - dependency-name: "HIPIFY"
+      - dependency-name: "rccl"
+      - dependency-name: "rccl-tests"
+      - dependency-name: "rocprof-trace-decoder"
+      - dependency-name: "rocm-cmake"
+      - dependency-name: "rocm-systems"
+    reviewers:
+      - "ScottTodd"
+      - "marbre"
```

Comment:
Fine to start as you have this now, but we should add a github team or some labels to help with review, rather than tag individual maintainers like this.